glide.yaml

```diff
@@ -281,3 +281,4 @@ import:
   version: v1.0
 - package: github.com/klauspost/cpuid
   version: v1.0
+- package: github.com/hashicorp/golang-lru
```

Comment:
It would be better if you could pin down the version here to whatever you need, to avoid automatic bumps. It looks like they are not tagging releases (yet), but you can pin to a specific commit.